app/assets/stylesheets/application.css

```diff
@@ -13,3 +13,114 @@
  *= require_tree .
  *= require_self
  */
+@import url('https://fonts.googleapis.com/css2?family=Roboto&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Unbounded&display=swap');
+
```

Comment:
- [ ] Please fix this error in your post model 
![image](https://user-images.githubusercontent.com/74744787/209442295-e2dd4f6a-da28-4f3f-9743-c20c20a9d1cd.png)